README.md

```
@@ -102,5 +102,213 @@ git commit -m "<Type>: <Message>";
 | `comment` | 주석만 수정 | 메소드 설명 주석 추가 |
 | `rename` | 파일/폴더 이름 변경 | UserService → MemberService |
 | `remove` | 파일 삭제 | 사용 안 하는 클래스 삭제 |
+
 <h2 align="center">7. 실행 예시</h2>
-이곳에 영상 첨부 예정
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)